tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/HotChocolateTags.cs

```diff
@@ -0,0 +1,12 @@
+// <copyright file="HotChocolateTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
+{
+    internal class HotChocolateTags : GraphQLTagsBase
+    {
+        public override string InstrumentationName => HotChocolateCommon.IntegrationName;
+    }
+}
```

Comment:
Rather than creating a base class, and overriding this (which creates completely seperate source-generated versions), I would have preferred to pass in `HotChocolateCommon.IntegrationName` into a constructor and set it there?

Reply:
👍 , the other day I suggested we do the same as for SQL Tags. ie having only one set of tag and setting the instrumentationname like this: https://github.com/DataDog/dd-trace-dotnet/blob/bf21db655abf35ff64ca46a6e1af5355f3c80401/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs#L48